src/Components/Publishing/ArticleMeta.tsx

```diff
@@ -0,0 +1,114 @@
+import { getFullEditorialHref } from "Components/Publishing/Constants"
+import { ArticleData } from "Components/Publishing/Typings"
+import React from "react"
+import { crop } from "Utils/resizer"
+
+export const ArticleMeta: React.SFC<{
+  article: ArticleData
+  customMetaContent: ArticleData
+  sd: any // res.locals.sd passed directly to component as sd
```

Comment:
Not sure what the best way to deal with `sd` values is in this context, I based this on an older stitch implementation used on the [artist page here](https://github.com/artsy/force/blob/89e6d5f9b7928ee312abc863fc7f585ede9e94d1/src/desktop/apps/artist/server.tsx#L62).

Reply:
There’s a few options.

* It looks like there’s only 2 values being used, perhaps those can just be distinct props without needing all of `sd` passed in?
* Are these values global (env) values or do they differ per request? If global, you can just expose types for them [here](https://github.com/artsy/reaction/blob/master/typings/sharify.d.ts) and then import sharify, rather than accepting it as a prop.

Reply:
I see that the `RESPONSIVE_CSS` value is the CSS generated by fresnel and currently _is_ being [generated per request](https://github.com/artsy/force/blob/5144face4cfa4859e111b87b22035521f88c60bc/src/desktop/apps/article/routes.tsx#L133), but it actually should only be generated once at the file level. Also, in case it is _only_ used by this meta component, perhaps it can just import that helper here removing the need to even pass it in from Force.